code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -580,3 +580,16 @@
 
 /mob/living/carbon/xenomorph/on_eord(turf/destination)
 	revive(TRUE)
+
+/mob/living/carbon/xenomorph/verb/upgrade_menu()
```

Comment:
doc, this being a verb is also bleh. should add a button to hive status (not priority, can tm without)